schema/2.0.0/corpora_schema.md

```diff
@@ -145,13 +145,13 @@ The following ontology dependencies are *pinned* for this version of the schema.
 
 cellxgene requires ENSEMBL gene identifiers to ensure that all datasets it stores measure the same features and can therefore be integrated.
 
-The following gene annotation dependencies are *pinned* for this version of the schema.
+The following gene annotation dependencies are *pinned* for this version of the schema. **Cells from any organism are allowed as long as orthologs from these annotations are used.**
 
 | Organism | Required version | Download |
 |:--|:--|:--|
 | [ENSEMBL (Human)] | Human reference GRCh38 (GENCODE v32/Ensembl 98)] matching [cellranger 2020-A (July 7, 2020) release] | [gencode.v32.primary_assembly.annotation.gtf] |
 | [ENSEMBL (Mouse)] | Mouse reference mm10 (GENCODE vM23/Ensembl 98) matching [cellranger 2020-A (July 7, 2020) release]| [gencode.vM23.primary_assembly.annotation.gtf] |
-|||
+| [ENSEMBL (COVID-19)] | SARS-CoV-2 reference (ENSEBML assembly: ASM985889v3) | [Sars\_cov\_2.ASM985889v3.101.gtf.gz] |
```

Comment:
(ENSEBML assembly: ASM985889v3) -> (**ENSEMBL** assembly: ASM985889v3)

Nit - but we should be consistent. I did not include the .gz in the label because I felt it was just "noise".

Reply:
fixed